### PR TITLE
Closes #1273 akutil.plotting to arkouda

### LINKED ISSUE #1273: `akutil.plotting` move to `arkouda`
Move `akutil.plotting` into arkouda. 

`akutil.hist` does not need to be moved because it is being moved to replace `ak.histogram` in #1272. 

This is being done because matplotlib is already required with pandas.

### PR DESCRIPTION
**Pull request by Ethan-DeBandi99**
This PR (closes #1273):

Plotting functionality has been moved to `arkouda/plotting.py`. Because `hist()` was moved to `arkouda/numeric.py` to update `ak.histogram()`, it has been removed from `arkouda/plotting.py`.

I was unsure of the best way to test the plotting. `matplotlib` has functionality to verify the resulting plots ([Documented Here](https://matplotlib.org/3.5.0/devel/testing.html)). This would require saving off and storing an example image though. If anyone has any suggestions on testing the plotting, please let me know.

### CODE REVIEW

**Comment by Ethan-DeBandi99** [comment]
Converting to draft until I can identify the issue with docs CI step

File: arkouda/plotting.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    >>> from matplotlib import pyplot as plt
+    >>> b, h = ak.histogram(ak.arange(10), 3)
+    >>> ak.plot_dist(b, h)
+    >>> # to show the plot
+    >>> plt.show()
+    """
+    if newfig:
+        plt.figure(figsize=(12, 5))
+    plt.subplot(1, 2, 1)
+    plt.plot(b, h, marker='.', linestyle='solid')
```
Comment (mhmerrill) [inline arkouda/plotting.py:46]: there is no explicit `h.to_ndarray()` what is up? does the example in the docstring work?

File: arkouda/plotting.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    if log:
+        plt.yscale('log')
+    if xlabel is not None:
+        plt.gca().set_xlabel(xlabel, fontsize=14)
+    plt.gca().set_title('distribution')
+    plt.subplot(1, 2, 2)
+    plt.plot(b, np.cumsum(h)/np.sum(h), marker=None, linestyle='solid')
```
Comment (mhmerrill) [inline arkouda/plotting.py:53]: same comment as above?!? maybe also use `ak.cumsum(h)/ak.sum(h)` since `h` is a `pdarray`

Comment (Ethan-DeBandi99) [reply arkouda/plotting.py:53]: See above.

File: arkouda/plotting.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import numpy as np  # type: ignore
+from matplotlib import pyplot as plt  # type: ignore
+
+from arkouda.numeric import log
+
+
+def log10(x):
+    basechange = float(np.log10(np.exp(1)))
```
Comment (mhmerrill) [inline arkouda/plotting.py:8]: i get it now... this should prob get a docstring... the key thing here is implementing `log10(x)` for a `pdarray`, see the imported `log` from `arkouda.numeric`... maybe we should `put `log10` into `arkouda.numeric`

**Review by mhmerrill — CHANGES_REQUESTED** [review]

File: arkouda/plotting.py (hunk 3 above)
Comment (mhmerrill) [reply arkouda/plotting.py:8]: maybe also `typeguard` it...

Comment (Ethan-DeBandi99) [reply arkouda/plotting.py]: @mhmerrill - I moved `log10` to `arkouda.numeric`. Docstring has been added as well as typegaurd.

**Comment by mhmerrill** [comment]
Looks good

**Review by ronawho — APPROVED** [review]
Approving on Mike's behalf

**Pull request merged**